lib/integration/notifyConnector.ts

```diff
@@ -48,3 +49,50 @@ export const getNotifyInstance = () => {
 
   return notifyInstance;
 };
+
+export const sendEmail = async (email: string, personalisation: Record<string, string>) => {
+  try {
+    const templateId = process.env.TEMPLATE_ID;
```

Comment:
So we allow the options to use a different template ID and default to this one?

Fine with leaving as is just asking questions

Reply:
I don't think we should include this option unless it's needed at the moment.